bodhi/server/services/releases.py

```diff
@@ -327,3 +346,55 @@ def save_release(request):
     request.db.flush()
 
     return r
+
+
+@critical_packages.put(schema=bodhi.server.schemas.UpdateCriticalPackages,
+                       renderer='json',
+                       error_handler=bodhi.server.services.errors.json_handler,
+                       validators=(colander_body_validator, validate_tags, validate_enums)
+                       )
```

Comment:
We need to be sure to use the admin ACL factory here so that only admins can edit the critpath packages.